app/models/admin_blacklisted_email.rb

```diff
@@ -0,0 +1,36 @@
+class AdminBlacklistedEmail < ActiveRecord::Base
+  attr_accessible :email
+
+  before_validation :canonicalize_email
+  
+  validates :email, presence: true, uniqueness: true, email_veracity: true
+
+  def canonicalize_email
+    self.email = AdminBlacklistedEmail.canonical_email(self.email) if self.email
+  end
+  
+  # Produces a canonical version of a given email reduced to its simplest form
+  # This is what we store in the db, so that if we subsequently check a submitted email,
+  #  we only need to clean the submitted email the same way and look for it.
+  def self.canonical_email(email_to_clean)
+    canonical_email = email_to_clean.downcase
+    canonical_email.strip!
+    canonical_email.sub!('@googlemail.','@gmail.')
```

Comment:
Actually no it wouldn't, it would only affect googlemail.catbus.co.uk. But anyway I'll put in the .com as that is the most common domain suffix and possibly the only working one now. I do point out, the goal here is to prevent trolls from successfully sending emails to a user who doesn't want them. We will only be entering email addresses that users write to us to ask to put into the blacklist. If a user complains of trolling or spam and they have created some weird email system for themselves or have an easily spoofed email address (more likely actually would be that they have a "catchall" email address where [anything]@userdomain.com goes to them) then ultimately we may have to tell them we can't stop people spoofing them.